packages/shared/src/hooks/usePageParams.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+import { NotificationPromptSource } from '../lib/analytics';
+import { useNotificationContext } from '../contexts/NotificationsContext';
+import { stripLinkParameters } from '../lib/links';
+
+export const usePageParams = (): void => {
```

Comment:
I do like the approach here, but now it's very focussed on a singular hook right?

I think we have two solutions:
1. Rename this to `useNotificationParams` (as it's only handling this for now)
2. Change the implementation so it can subscribe to multiple elements and handle them accordingly (example is squad creation and auth having these in place already)

Let me know what you think?
(My suggestion would be option 1 for now, we can abstract for another one?)

Reply:
That makes sense, better to be clear than vague especially it does only one thing for now. Let's go with option 1.